tests/integration/utils.py

```diff
@@ -136,6 +137,18 @@ def fetch_content_manifest(self, request_id):
         resp.raise_for_status()
         return Response(resp.json(), request_id, resp.status_code)
 
+    def fetch_sbom(self, request_ids):
+        """
+        Fetch a sbom for request_ids  from the Cachito API.
+
+        :param str request_ids: The IDs of the Cachito requests separated by ','
```

Comment:
For consistency, could the param here be just a single request_id?

The way it's used in the assert_sbom method, we're passing an `int` anyway

```suggestion
    def fetch_sbom(self, request_id):
        """
        Fetch a sbom for request_id  from the Cachito API.

        :param int request_id: The ID of the Cachito request
```

Reply:
this test is testing multiple requests so it is used:
tests/integration/test_content_manifest.py

https://github.com/containerbuildsystem/cachito/pull/810/files#diff-d7e6e90e53263b19df6852014b3781b35f20d611aa02892d305f12b9162fff1aR76

Reply:
Ah, ok. Could you instead update the assert_sbom method to pass a string rather than an integer?

Reply:
done